server/@types/viewModels/index.d.ts

```diff
@@ -81,7 +81,7 @@ declare module 'viewModels' {
   }
 
   /**
-   * A prisoner's record of In Prison courses and eduction, which is made up of a collection of InPrisonEducation.
+   * A prisoner's record of In-Prison courses and eduction, which is made up of a collection of InPrisonEducation.
    * TODO - needs better name and field definition once we understand the data and the requirements.
```

Comment:
@mattwills-moj-digital @chrimesdev - What do we think of the names `InPrisonEducationRecords` and `InPrisonEducation` as type names in our view model?
If we are happy with the names I'll get rid of the TODOs in this PR; else do we have any better name suggestions?; else I'll leave it for another day
What do you think?

Reply:
Those names seem OK to me, without looking at the code and seeing the names first - I had an good idea of what to expect from them!

Reply:
I think it's ok too. An alternative might be `InPrisonEducation` which consists of a collection of `InPrisonCourse`, but I'm not sure if that's any better? I'll let you choose 😄

Reply:
I'll leave it for now; but will remove the TODOs
(mainly because changing the name would mean changing all the test data builders, variables that use test data instances etc etc)
😁

Reply:
No problem, I might change it myself! 😂